modules/consul-cluster/variables.tf

```
@@ -164,6 +164,11 @@ variable "root_volume_delete_on_termination" {
   default     = true
 }
 
+variable "root_volume_encrypted" {
+  description = "Encrypt the root volume at rest"
+  default     = false
```

Crap, sorry, one more thing I missed on the first go around:

```suggestion
  description = "Encrypt the root volume at rest"
  type        = bool
  default     = false
```